sdk/keyvault/keyvault-secrets/src/secretsModels.ts

```diff
@@ -83,8 +83,18 @@ export interface SecretProperties {
    * this field specifies the corresponding key backing the KV certificate.
    * **NOTE: This property will not be serialized. It can only be populated by
    * the server.**
+   * @deprecated Please use {@link SecretProperties.certificateKeyId} instead. This field will always be undefined.
```

Comment:
Nit: Some may take the "this" here to refer to `certificateKeyId` :)

How about the below?

> The `keyId` field has and will be left as undefined.